app/proguard-rules.pro

```diff
@@ -1,20 +1,3 @@
--keep class com.simplemobiletools.** { *; }
--dontwarn android.graphics.Canvas
--dontwarn com.simplemobiletools.**
--dontwarn org.apache.**
-
-# Picasso
--dontwarn javax.annotation.**
--keepnames class okhttp3.internal.publicsuffix.PublicSuffixDatabase
--dontwarn org.codehaus.mojo.animal_sniffer.*
--dontwarn okhttp3.internal.platform.ConscryptPlatform
-
--keepclassmembers class * implements android.os.Parcelable {
-    static ** CREATOR;
-}
-
-# RenderScript
--keepclasseswithmembernames class * {
-native <methods>;
-}
--keep class androidx.renderscript.** { *; }
```

Comment:
t+These rules are not necessary because they are either included in `proguard-android-optimize.txt` or the libraries themself.